addon/components/date-range-picker.js

```diff
@@ -80,6 +84,9 @@ export default Ember.Component.extend({
     this.setupPicker();
   },
 
+  isInvalidDate: function(){},
+  isCustomDate: function(){},
```

Comment:
Hi @josemarluedke yeah, it does nothing by default, but if you pass a function it will act as a filter function for each day on the calendar. I don't quite remember the exact behavior but `IsInvalidDate` expects true a false (like a filter) and `isCustomDate` lets you return a custom string (a class name) for every date.